workflows/data_pipelines/finess/config.py

```diff
@@ -0,0 +1,17 @@
+from dag_datalake_sirene.config import (
+    DataSourceConfig,
+    BASE_TMP_FOLDER,
+    DATA_GOUV_BASE_URL,
+    MINIO_BASE_URL,
+)
+
+FINESS_CONFIG = DataSourceConfig(
+    name="finess",
+    tmp_folder=f"{BASE_TMP_FOLDER}/finess",
```

Comment:
nit: Since all those variables are used everywhere we could directly set them up in `DataSourceConfig` so we could do something like:
```
FINESS_CONFIG = DataSourceConfig(
    name="finess",
    tmp_folder=f"{DataSourceConfig.base_tmp_folder}/finess",
    ...
)
```
With:
```
@dataclass
class DataSourceConfig:
    name: str
    tmp_folder: str
    minio_path: str
    file_name: str
    base_tmp_folder: str = "/tmp"
    resource_id: str | None = None
    url: str | None = None
    url_minio: str | None = None
    url_minio_metadata: str | None = None
```
This would reduce the imports copy pastes.
What do you think?

Reply:
Btw the config.py is so small. Should we define it directly in the DAG file ? 🤔

Reply:
It's a good idea, I kept it out because we might define it at the final refacto stage in an AirflowConfig class. But I'll add it now and we'll cross that bridge when we come to it.

Reply:
From a Separation of Concerns perspective, I believe it’s better to keep the configuration separate.

As we progress deeper into the refactor, the configuration is likely to evolve and might eventually be relocated. 

Additionally, since this configuration is also used in the ETL DAG, it feels cleaner and more maintainable to import it from a dedicated configuration file rather than embedding it directly in the DAG file.

 What are your thoughts?

Reply:
I will merge and we can discuss the config issue on Friday if that's okay with you :)

Reply:
Sure thing! We can definitely wait for the config class to be more mature before raising the point again.
It's nice to see the refactor project going forward!